prometheus.yml

```
@@ -0,0 +1,166 @@
+  external_labels:
+    environment: 'production'
+    application: 'super-blue-badge-app'
+
+rule_files:
+  - "rules/*.yml"
+
+alerting:
+  alertmanagers:
+    - static_configs:
+        - targets:
+          - alertmanager:9093
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 12 but found 10

(indentation)